lib/beaker/hypervisor/vagrant.rb

```diff
@@ -91,7 +91,7 @@ def make_vfile(hosts, options = {})
           end
         end
 
-        if /windows/i.match(host['platform'])
+        if /windows/i.match?((host['platform']))
```

Comment:
These braces look redundant. Also repeated a few times

Reply:
fixed it, looks good now.